book/P2C9_calib.qmd

```diff
@@ -67,12 +67,12 @@ A model is well-calibrated with respect to $H_\alpha$ if $H_\alpha = 1$.
 
 The next metrics we look at evaluate models across a spectrum of points to assess calibration over time.
 
-## Probabilistic Calibration {#sec-eval-distr-calib-prob}
+## Probabilistic Calibration {#sec-calib-prob}
 
 Calibration over a range of time points may be assessed quantitatively or qualitatively, with graphical methods often favoured.
 Graphical methods compare the average predicted distribution to the expected distribution, which can be estimated with the Kaplan-Meier curve, discussed next.
 
-### Kaplan-Meier Comparison
+### Kaplan-Meier Comparison {#sec-calib-km}
```

Comment:
I think the message here is that CPH and KM are equally well calibrated. But since only calibration is measured, no telling how well they discriminate. Which is obvious since the KM is used as bench mark, which doesn't discriminate at all

Reply:
Let's discuss, I don't think that is the message here. This is a calibration method, not commenting at all on discrimination. KM is always well-calibrated *because* it doesn't discriminate.